src/main/resources/messages/messages.properties

```diff
@@ -609,6 +618,9 @@ error.distribution.empty_file=The '--file' value can not be empty for the 'defau
 error.distribution.empty_bundle_ids=The '--bundle-id' value can not be empty for the 'bundle' export mode
 error.distribution.incorrect_file_command_usage=The '--file' is used only for the 'default' export mode
 error.distribution.incorrect_bundle_id_command_usage=The '--bundle-id' is used only for the 'bundle' export mode
+error.distribution.strings_based_export_mode=String-based projects do not support export mode setting
```

Comment:
Perhaps it would be also nice to have such a check in the `distribution add` command as well. What do you think?